### PR TITLE
[update] update network podRecords

### PR DESCRIPTION
**Pull request by fewdan**
Signed-off-by: “fewdan” <fewdan@hotmail.com>

### What problem does this PR solve?

https://github.com/chaos-mesh/chaos-mesh/issues/1114

### What is changed and how does it work?
If a pod appears in both source and target, it will appear repeatedly in pod records.
Make a distinction in the message.

### Checklist

Tests

- [ ] Unit test
- [ ] E2E test
- [ ] Manual test (add detailed scripts or steps below)
- [ ] No code

Side effects

- [ ] Breaking backward compatibility

Related changes

- [ ] Need to update the documentation

### Does this PR introduce a user-facing change?

```release-note
NONE
```

### CODE REVIEW

File: controllers/networkchaos/trafficcontrol/types.go [hunk 1]
```diff
@@ -132,6 +134,12 @@ func (r *endpoint) Apply(ctx context.Context, req ctrl.Request, chaos v1alpha1.I
...
 		}
 
+		if index < len(sources) {
+			ps.Message = networkChaosSourceMsg
+		} else {
+			ps.Message = networkChaosTargetMsg
+		}
+
 		if networkchaos.Spec.Duration != nil {
 			ps.Message = fmt.Sprintf(networkTcActionMsg, *networkchaos.Spec.Duration)
```
Comment (WangXiangUSTC) [inline controllers/networkchaos/trafficcontrol/types.go:144]: use `+=`?

**Review by cwen0 — APPROVED** [review]
LGTM

**Review by WangXiangUSTC — APPROVED** [review]
LGTM

**Comment by fewdan** [comment]
I will submit a pr to a separate cherry-pick.

**Pull request merged**